extensions/bitwarden/src/utils.ts

```diff
@@ -65,3 +66,29 @@ export function extractKeywords(item: Item): string[] {
   const uniqueKeywords = new Set(keywords.filter((keyword): keyword is string => !!keyword));
   return [...uniqueKeywords];
 }
+
+/**
+ * Runs an AppleScript script.
+ *
+ * @param applescript The script code.
+ * @param args The arguments to pass to the script.
+ */
+export function runApplescript(applescript: string, args: string[]): Promise<string> {
+  return new Promise((resolve, reject) => {
+    execFile(
+      "/usr/bin/osascript" /* hardcoded for security */,
+      ["-e", applescript, ...args],
+      {
+        shell: false,
+      },
+      (err, stdout, _stderr) => {
+        if (err != null) {
+          reject(err);
+          return;
+        }
+
+        resolve(stdout);
+      }
+    );
+  });
+}
```

Comment:
Small suggestion here, we could make use of the `execa` library we have as a dependency and use the `async await` syntax.

Don't forget to delete the import statement.

```suggestion
export async function runAppleScript(appleScript: string, args: string[]) {
  const { stdout } = await execa("/usr/bin/osascript", ["-e", appleScript, ...args], { shell: false });
  return stdout;
}
```